README.md

```diff
@@ -0,0 +1,18 @@
+# Google Analytics plugin for Snowplow
+
+## Overview
+
+[sp-ga-plugin.js](sp-ga-plugin.js) lets you fork the payloads sent to Google Analytics to your
+Snowplow endpoint.
+
+You can use the plugin by requiring it and specifying your Snowplow endpoint:
```

Comment:
Please move this usage into a dedicated new section in the snowplow/snowplow wiki